coderd/workspaceapps_test.go

```diff
@@ -262,6 +262,13 @@ func TestWorkspaceApps(t *testing.T) {
 			opts.AppHost = ""
 		}
 
+		statsFlushCh := make(chan chan<- struct{})
+		opts.StatsCollectorOptions.Flush = statsFlushCh
+		flushStats := func() {
+			flushDone := make(chan struct{})
+			statsFlushCh <- flushDone
+			<-flushDone
+		}
```

Comment:
I see you removed the channel buffering from both channels, any particular reason? I suppose it can work without them, but I try to avoid those so there's no risk of deadlocks. 😄

Reply:
I'll put back the buffers; I was messing around with a few different things.